benchmarks/dynamo/common.py

```diff
@@ -1393,7 +1410,31 @@ def parse_args(args=None):
     parser.add_argument(
         "--fast", "-f", action="store_true", help="skip slow benchmarks"
     )
-    parser.add_argument("--only", help="Run just one model")
+    parser.add_argument(
```

Comment:
are you landing this separately?

Reply:
yep, this is supposed to be landed by #89028 . But that PR is blocked by some 'Mac OS' CI issue which affects multiple PRs.